src/frontend/mame/luaengine_mem.cpp

```diff
@@ -652,9 +538,9 @@ void lua_engine::initialize_memory()
  */
 
 	auto memory_type = sol().registry().new_usertype<memory_manager>("memory", sol::no_constructor);
-	memory_type["banks"] = sol::property([] (memory_manager &mm) { return standard_memory_object_map<memory_bank>(mm.banks()); });
-	memory_type["regions"] = sol::property([] (memory_manager &mm) { return standard_memory_object_map<memory_region>(mm.regions()); });
-	memory_type["shares"] = sol::property([] (memory_manager &mm) { return standard_memory_object_map<memory_share>(mm.shares()); });
+	memory_type["banks"] = sol::property([] (memory_manager &mm) { return lua_engine::standard_tag_object_ptr_map<memory_bank>(mm.banks()); });
+	memory_type["regions"] = sol::property([] (memory_manager &mm) { return lua_engine::standard_tag_object_ptr_map<memory_region>(mm.regions()); });
+	memory_type["shares"] = sol::property([] (memory_manager &mm) { return lua_engine::standard_tag_object_ptr_map<memory_share>(mm.shares()); });
```

Comment:
These shouldn’t need `lua_engine::` since `standard_tag_object_ptr_map` is not an argument-dependent name.